CMakePresets.json

```diff
@@ -26,7 +26,7 @@
     },
     {
       "name": "imcs_workstation_RELWITHDEBINFO",
```

Comment:
@mayrmt I am curious why not name it simply "imcs_workstation_debug"?

Reply:
Because this is not a debug build. This is a release build with debug information. There is a difference in compiler optimization. If we want to have an actual debug build, we need to add it as a separate build.